### DISCUSSION TITLE
Could kind2 run on windows without WSL?

### DISCUSSION THREAD

**Discussion by kumori123** [kind2-mc/kind2#933] [Q&A] [answered]
Could kind2 run on windows without WSL?
Thank you!

**Comment by daniel-larraz** [comment, marked as answer]
No, Kind 2 does not run on Windows natively. Alternatively, you can use [Docker](https://github.com/kind2-mc/kind2#docker) to run Kind 2 on Windows.

Best,
Daniel

**Comment by kumori123** [comment]
Hello, I found that Kind2 can run with the help of WSL2. But I failed to run it on some common virtual linux environment for windows like crygin, msys2. Do you know the reason?
Thank you!

**Reply by daniel-larraz** [reply]
You cannot use Cygwin or MSYS2 to run native Linux binaries on Windows. You must rebuild Kind 2 from source if you want it to run on Windows. Unfortunately, there is no easy way to do that currently, and we don't have any cycles to work on it.
